README.md

```diff
@@ -191,13 +191,44 @@ Install `cmake` and `go`:
 brew install cmake go
 ```
 
-Then generate dependencies and build:
+Then generate dependencies:
+```
+go generate ./...
+```
+Then build the binary:
+```
+go build .
+```
+
+### Linux/Windows CUDA (NVIDIA)
```

Comment:
I think we should move these extended docs down to `docs/development.md` and link to them from the main README